src/components/action-drawer/ActionDrawer.tsx

```diff
@@ -50,12 +51,16 @@ const ActionDrawer = ({
           </div>
           <ul aria-labelledby={headerId}>
             {actions.map(
-              ({ text, isDestructive = false, className, icon }, index) => (
+              (
+                { text, isDestructive = false, className, icon, onClick },
+                index
+              ) => (
                 <li
                   key={text}
                   role='button'
                   tabIndex={0}
-                  onClick={() => {
+                  onClick={event => {
```

Comment:
I like this way more. We already do this in the mobile client ActionDrawer too:

https://github.com/AudiusProject/audius-mobile-client/blob/master/src/components/action-drawer/ActionDrawer.tsx#L17